Framework/PythonInterface/mantid/utils/nomad/_median_detector_test.py

```diff
@@ -0,0 +1,298 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+# 3rd-party imports
+import numpy
+import numpy as np
+import yaml
+from mantid.simpleapi import (LoadEmptyInstrument, mtd, MaskDetectors, ExtractMask, SaveMask, DeleteWorkspace)
+# standard imports
+import enum
+from collections import namedtuple
+
+
+__all__ = ['determine_tubes_threshold']
+
+
+def determine_tubes_threshold(vec_intensity, mask_config: dict, instrument_config):
```

Comment:
any particular reason we are doing partial type hinting here?

Reply:
This file's going to go. I'm very sorry I forgot to remove it